README.md

```diff
@@ -1,21 +1,30 @@
-# Trailblazer Adapters
+# Trailblazer Whitelisted Protocols
 
-## Adding a new protocol adapter
+## Adding a New Protocol to the Whitelist
 
-### 1. Add your project under `adapters/projects`
+### 1. Modify `protocols.json`
 
-### 2. Write an indexer in the new folder
+Open [`protocols.json`](./whitelist/protocols.json) and add a new protocol entry to the whitelist.
 
-An example adapter for tracking OrderFulfilled Event on the OKX marketplace can be seen [here](./adapters/projects/okx/order_fulfilled.go)
+### 2. Example Protocol Entry
 
-### 3. Test out the adapter by adding it to the cli
+A new protocol entry needs atleast a name, slug and its contract addresses. Optionally, add twitter and logo reference. Logo must be stored under `/img`.
 
-An example on how to add a new adapter to the cli is [here](./cmd/README.md)
+```json
+{
+    "name": "Name of Protocol",
+    "slug": "name-of-protocol",
+    "contracts": [
+        "0x0"
+        ...
+    ]
+}
+```
 
-### 4. Create a Pull Request
+### 3. Create a Pull Request
 
 Create a pull request (PR) on GitHub to merge your changes into the main branch. Provide a clear description of the changes and the protocol added.
 
-### 5. Review and Merge
+### 6. Review and Merge
```

Comment:
Bulletpoint number `4.`, also change in same README under `whitelist/README.md`.